pkg/controller/middleware/mfa.go

```diff
@@ -40,8 +41,14 @@ func RequireMFA(ctx context.Context, h *render.Renderer) mux.MiddlewareFunc {
 				return
 			}
 
+			user := controller.UserFromContext(ctx)
```

Comment:
Nit: after doing the auditing stuff, I'd like us to standardize on `currentUser` when pulling from the context.